hyperSpec/R/Colors.R

```diff
@@ -0,0 +1,31 @@
+#'
+#' Color Suggestions for `hyperSpec` Users
+#'
+#' In `hyperSpec` the user may use any color name/format known to `R`.
+#' An important fact to keep in mind is that most people with normal vision cannot distinguish
+#' more than about 8-12 colors, and doing so depends upon the viewing circumstances:
+#' if on paper, printer, ink and paper type all matter, and if on a screen, the background color
+#' makes a big difference.  Further, color-blind individuals have additional challenges.
+#' A great discussion of color issues can be found in the `colorspace` package.  The
+#' `Polychrome` package has further discussion and utilities for choosing qualitative
+#' colorschemes, including those for color-blind individuals.
+#'
+#' `CB7` is a selection of seven colors suitable for use on screens by people with 
```

Comment:
The object's name `CB7` is short, but it is not informative and contains upper-case letters so does not adhere to the **tidyverse** style guide: https://style.tidyverse.org/syntax.html#object-names

I recommend renaming it into something more intuitive like `colorblind_palette_7`. In the examples, it would get easier to understand that a color palette is used without the need to browse in some additional pages of documentation.

Reply:
Sure, though I don't find the `tidyverse` quite as compelling as you do.

Reply:
In `CONTRIBUTING.md`:

![image](https://user-images.githubusercontent.com/12725868/88002132-6dc59c00-cb0a-11ea-9b68-51ac05ce6408.png)

Reply:
I'm going to leave this for now.  When first used its purpose is given.  A longer name will make some code sections quite a bit longer.